tests/app/helpers/test_template_helpers.py

```diff
@@ -216,6 +217,16 @@ def test_footer_warning_not_in_context_census_theme(app: Flask):
                 read_file("./templates/assets/images/ni-finance-mobile-logo.svg"),
             ],
         ),
+        (
+            None,
```

Comment:
Should we use the type here? 
```suggestion
            SurveyType.BEIS,
```

Reply:
I was following the NI suff when doing this, does this mean you think we should add the equivalent for NI?

Reply:
Just not clear to me why either are `None`, the normal business config seems to have a test for both